libraries/build.gradle.kts

```diff
@@ -0,0 +1,3 @@
+plugins {
```

Comment:
interesting! didn't realise this works applies to all the child modules!

Reply:
Does this then cause this to become a gradle module? (I'm unsure if it was already considered that way before)